content/en/sensitive_data_scanner/setup/telemetry_data.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Telemetry Data
+disable_toc: false
+further_reading:
+  - link: "/sensitive_data_scanner/scanning_rules/library_rules"
+    tag: "Documentation"
+    text: "Learn more about out-of-the-box library rules"
+  - link: "/sensitive_data_scanner/scanning_rules/custom_rules"
+    tag: "Documentation"
+    text: "Learn more about creating custom rules"
+---
+
+## Overview
+
+Sensitive Data Scanner in the Cloud scans telemetry data, such as your application logs, APM events, RUM events, and events from Event Management. The data that can be scanned and redacted are:
+
+- Logs: All structured and unstructured log content, including log message and attribute values
+- APM: Span attribute values only
+- RUM: Event attribute values only
+- Events: Event attribute values only
+
+You submit logs and events to the Datadog backend, so the data leaves your environment before it gets redacted. However, the logs and events are scanned and redacted in the Datadog backend during processing, so sensitive data is redacted before events are indexed in Datadog.
+
+If you don't want data to leave your environment before it gets redacted, use [Observability Pipelines][12] and the [Sensitive Data Scanner processor][13] to scan and redact sensitive data. See [Set Up Pipelines][14] for information on how to set up a pipeline and its components.
+
+To use Sensitive Data Scanner in the Cloud, set up a scanning group to define what data to scan and then add scanning rules to determine what sensitive information to match within the data.
+
+This document goes through the following:
+
+- The [permissions](#permissions) required to view and set up Sensitive Data Scanner.
+- [Adding a scanning group](#add-a-scanning-group)
+- [Adding scanning rules](#add-scanning-rules)
+- [How to control access to logs wth sensitive data](#control-access-to-logs-with-sensitive-data)
+- [How to redact sensitive data in tags](#redact-sensitive-data-in-tags)
+
+## Setup
```

Comment:
The rest of the content on this page was moved from the current landing pages and didn't have many, if any, updates.

Reply:
👍🏻 I skimmed through it and spot checked the links. Looks good!